unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldUtils.cs

```diff
@@ -94,5 +100,49 @@ public static bool IsPointerOverUIElement()
         return results.Count > 1;
     }
 
+    public static string ConvertEntityToJSON(DecentralandEntity entity)
+    {
+        BuilderInWorldEntityData builderInWorldEntityData = new BuilderInWorldEntityData();
+        builderInWorldEntityData.entityId = entity.entityId;
+
+
+        foreach (KeyValuePair<CLASS_ID_COMPONENT, BaseComponent> keyValuePair in entity.components)
+        {
+            if (keyValuePair.Key == CLASS_ID_COMPONENT.TRANSFORM)
```

Comment:
As stated before, this wont scale if the amount of components increase. We can leave it for when the need arises, but let's start thinking about a it!